platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/hashmap/HalfDiskHashMap.java

```diff
@@ -318,8 +318,8 @@ public LongSummaryStatistics getFilesSizeStatistics() {
      */
     @Override
     public void close() throws IOException {
-        bucketIndexToBucketLocation.close();
         fileCollection.close();
+        bucketIndexToBucketLocation.close();
```

Comment:
If the order of the calls really matters that much, it might be worth adding a comment in code mentioning why.

Reply:
OK